core/src/test/kotlin/org/stellar/anchor/sep31/Sep31ServiceTest.kt

```diff
@@ -743,7 +737,7 @@ class Sep31ServiceTest {
     val wantTx =
```

Comment:
`memo` in L735 is assigned twice.

Reply:
In https://github.com/stellar/java-stellar-anchor-sdk/pull/1446/files#diff-4e0835ca8055b8e40053d5f39ecebc0b44ac62e037db1367d0d3d639949a4dd3L4

Can we use a different package?

Reply:
> memo in L735 is assigned twice.

Nice catch, this variable is no longer used anymore. I deleted it.

Reply:
> In https://github.com/stellar/java-stellar-anchor-sdk/pull/1446/files#diff-4e0835ca8055b8e40053d5f39ecebc0b44ac62e037db1367d0d3d639949a4dd3L4
>
> Can we use a different package?

The link doesn't seem to work. Can you clarify what you mean?

Reply:
ah... I cannot recall. It is minor. May be will catch next time.